src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="#" 
+      method="get" 
+      data-qa="big"
+    >
+      <div class="search-box">
+        <img 
+          class="search" 
+          src="./images/Search.svg" 
+          alt="#"
```

Comment:
Provide a meaningful alt attribute value for the search image, such as "Search Icon" 🧐